### PR TITLE
Remove deprecated image methods

### LINKED ISSUE #4309: Image-related deprecations
### Description

https://github.com/contao/contao/blob/d7ccb031e5f89e560cc6d1bd30b420f308891a14/core-bundle/src/Image/ImageFactory.php#L290

https://github.com/contao/contao/blob/d7ccb031e5f89e560cc6d1bd30b420f308891a14/core-bundle/src/Image/ImageSizes.php#L120

https://github.com/contao/contao/blob/d7ccb031e5f89e560cc6d1bd30b420f308891a14/core-bundle/src/Image/LegacyResizer.php#L49

### PR DESCRIPTION
**Pull request by ausi**
See #4309

### CODE REVIEW

File: core-bundle/src/Resources/config/services.yml [hunk 1]
```diff
@@ -1078,13 +1077,13 @@ services:
             version: 4.13
             message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "contao.image.sizes" instead.
 
-    contao.image.resizer:
-        alias: contao.image.legacy_resizer
+    contao.image.legacy_resizer:
```
Comment (leofeyer) [inline core-bundle/src/Resources/config/services.yml]: Why keep this at all?

File: core-bundle/src/Resources/contao/widgets/FileSelector.php [hunk 2]
```diff
@@ -0,0 +1,670 @@
...
+ * @property boolean $files
+ * @property boolean $filesOnly
+ * @property string  $extensions
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0.
+ *             Use the picker instead.
+ */
+class FileSelector extends Widget
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/widgets/FileSelector.php]: Did you accidentally add this class again?

File: core-bundle/src/Resources/config/services.yml (hunk 1 above)
Comment (ausi) [reply core-bundle/src/Resources/config/services.yml]: For BC? This service is pretty probable to get decorated IMO.

File: core-bundle/src/Resources/contao/widgets/FileSelector.php (hunk 2 above)
Comment (ausi) [reply core-bundle/src/Resources/contao/widgets/FileSelector.php]: Yes 🙈

Comment (ausi) [reply core-bundle/src/Resources/contao/widgets/FileSelector.php]: Fixed in b06a6f671e9a0fe11d079540dfb232b67f6d64ae

File: core-bundle/src/Resources/config/services.yml (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Resources/config/services.yml]: But we are allowed to break BC in Contao 5.0. Still want to keep the old service ID?

Comment (ausi) [reply core-bundle/src/Resources/config/services.yml]: I’d vote for keeping the old service ID. But I’m fine either way.

Comment (leofeyer) [reply core-bundle/src/Resources/config/services.yml]: I‘d vote for removing it (and possibly adding it again if this should cause issues). @contao/developers WDYT?

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**